scipy/spatial/transform/rotation.py

```diff
@@ -209,10 +210,8 @@ def __init__(self, quat, normalized=False):
             quat = quat[None, :]
             self._single = True
 
-        if normalized:
-            self._quat = quat
-        else:
-            self._quat = quat.copy()
+        self._quat = quat.copy()
```

Comment:
I suggest `copy=True` by default. For `normalized=False` we always copy, and we want to minimize user errors when `normalized=True` so I think we should copy the quaternions by default unless otherwise specified.